### ISSUE TITLE
Do we want a `--follow` option for `brig event create`

### ISSUE THREAD

**Issue by krancour** [brigadecore/brigade#1329]
v2 has a much stronger emphasis than v1.x on events being handled _asynchronously_, subject to scheduling constraints.

Because of this `brig run` which sounds like an imperative to "do it now," was replaced with `brig event create`.

While I agree with that change, I regardless find myself _frequently_ doing this:

```console
$ brig event create -p pipeline-demo
$ brig event logs -f -i <event uuid returned by previous command>
```

I think we _should_ entertain the idea of supporting a `--follow` (alias `-f`) flag on the `brig event create` command. It can either stream event status from the API or query event status periodically (with a progressive backoff?) until the event has advanced past a pending state. At that point, it can begin streaming even logs from the API.

I think such a flag would be a nice convenience and, provided it is documented properly, won't imply to the user that events are handled synchronously.

Anyone have thoughts about this?

**Comment by krancour** [comment]
Kicking this to beta since I believe this can be done with no API changes.

**Comment by krancour** [comment]
This was closed by #1373
